tests/ga4gh/registry/test_server.py

```diff
@@ -96,9 +97,21 @@ def test_getServiceTypes():
     app.config['FOCA'] = Config(
         db=MongoConfig(**MONGO_CONFIG)
     )
+
+    # write a couple of services into DB
+    app.config['FOCA'].db.dbs['serviceStore'].collections['services'] \
+        .client = mongomock.MongoClient().db.collection
+
+    for i in ["serv1", "serv2", "serv3"]:
+        mock_resp = deepcopy(MOCK_SERVICE)
+        mock_resp['id'] = i
+        app.config['FOCA'].db.dbs['serviceStore'].collections['services'] \
+            .client.insert_one(mock_resp)
+
     with app.app_context():
         res = getServiceTypes.__wrapped__()
-        assert res == []
+        # All written services have same type, we expect list of length 1
```

Comment:
Is this test enough or do we need to test for different types?

Reply:
Was first leaning to no, but now leaning to yes. Could be that someone refactors the controller in a way that it, e.g., always only returns a list of length 1 (e.g., the first or last service). We wouldn't have a test to catch this then. So yeah, would be good to have a test where two different services are returned. So can just copy this test and change the org or artifact for each, then check that the length of the returned services equals the length of the input list.